target/linux/ramips/base-files/etc/diag.sh

```diff
@@ -124,6 +124,7 @@ get_status_led() {
 	d240|\
 	dap-1350|\
 	na930|\
+	newifi-d2|\
```

Comment:
It is for sure at the wrong location. the board doesn't have a newifi-d2:blue:power led.